lib/dfe/reference_data/bank_holidays.rb

```diff
@@ -0,0 +1,131 @@
+module DfE
+  module ReferenceData
+    module BankHolidays
+      BANK_HOLIDAYS_SCHEMA = {
+        id: :string,
+        title: :string,
+        date: :datetime,
+        notes: { kind: :optional, schema: :string }
+      }.freeze
+
+      BANK_HOLIDAYS_FIELD_DEFINITIONS = {
+        id: 'A unique identifier',
+        title: 'The title of the bank holiday',
+        date: 'The date of the bank holiday',
+        notes: 'Notes about the bank holiday'
+      }.freeze
+
+      # N.B. England and Wales only
+      BANK_HOLIDAYS = DfE::ReferenceData::HardcodedReferenceList.new(
```

Comment:
Please can we also add in 2022 bank holidays?

Reply:
Now added 👍